conf/alembic.ini

```diff
@@ -0,0 +1,37 @@
+[alembic]
+script_location = lti:migrations
+sqlalchemy.url: postgresql://postgres@localhost:5433/postgres
```

Comment:
This is the default location for the DB in a dev env, it's overwritten by the `DATABASE_URL` env var in `env.py` below.